spec/features/invite_user_spec.rb

```diff
@@ -48,6 +48,9 @@ def register_with_valid_card(invitation={})
     select(valid_card[:exp_month], :from => 'exp_month' )
     select(valid_card[:exp_year], :from => 'exp_year' )
     click_button 'Sign Up'
+
+    sleep 3.0 # wait for the filling things done avoid random fails with capybara-webkit
```

Comment:
Adding `Capybara.default_max_wait_time = 5` on `spec_helper.rb` and remove the manual setting in test cases.